[Kaptain]: MinIO -> changing notes and flagging tar files (#4482)
mesosphere/dcos-docs-site: pages/dkp/kaptain/1.2.0/download/index.md

If you already have a license, downloads are available in the [Support Portal][support-portal].

> This Docker image includes code from the MinIO Project (“MinIO”), which is © 2015-2021 MinIO, Inc. MinIO is made available subject to the terms and conditions of the [GNU Affero General Public License 3.0](https://www.gnu.org/licenses/agpl-3.0.en.html). The complete source code for the versions of MinIO packaged with Kaptain 1.2.0 are available at this URL: [https://github.com/minio/minio/tree/RELEASE.2020-12-03T05-49-24Z](https://github.com/minio/minio/tree/RELEASE.2020-12-03T05-49-24Z)

Reviewer: This one is a Kudo package. :-)

```suggestion
This Kudo package image includes code from the MinIO Project (“MinIO”), which is © 2015-2021 MinIO, Inc. MinIO is made available subject to the terms and conditions of the [GNU Affero General Public License 3.0](https://www.gnu.org/licenses/agpl-3.0.en.html). The complete source code for the versions of MinIO packaged with Kaptain 1.2.0 are available at this URL: [https://github.com/minio/minio/tree/RELEASE.2020-12-03T05-49-24Z](https://github.com/minio/minio/tree/RELEASE.2020-12-03T05-49-24Z)
```

> This Docker image includes code from the MinIO Project (“MinIO”), which is © 2015-2021 MinIO, Inc. MinIO is made available subject to the terms and conditions of the [GNU Affero General Public License 3.0](https://www.gnu.org/licenses/agpl-3.0.en.html). The complete source code for the versions of MinIO packaged with Kaptain 1.2.0 are available at this URL: [https://github.com/minio/minio/tree/RELEASE.2020-12-03T05-49-24Z](https://github.com/minio/minio/tree/RELEASE.2020-12-03T05-49-24Z)

Author: ```suggestion
This Docker image includes code from the MinIO Project (“MinIO”), which is © 2015-2021 MinIO, Inc. MinIO is made available subject to the terms and conditions of the GNU Affero General Public License 3.0. The complete source code for the versions of MinIO packaged with Kaptain 1.2.0 are available at these URLs:
[https://github.com/minio/operator/releases/tag/v4.0.3](https://github.com/minio/operator/releases/tag/v4.0.3)
[https://github.com/minio/mc/releases/tag/RELEASE.2021-03-23T05-46-11Z](https://github.com/minio/mc/releases/tag/RELEASE.2021-03-23T05-46-11Z)
```

If you already have a license, downloads are available in the [Support Portal][support-portal].

> This Docker image includes code from the MinIO Project (“MinIO”), which is © 2015-2021 MinIO, Inc. MinIO is made available subject to the terms and conditions of the [GNU Affero General Public License 3.0](https://www.gnu.org/licenses/agpl-3.0.en.html). The complete source code for the versions of MinIO packaged with Kaptain 1.2.0 are available at these URLs:

Reviewer: ```suggestion
This Kudo package includes code from the MinIO Project (“MinIO”), which is © 2015-2021 MinIO, Inc. MinIO is made available subject to the terms and conditions of the [GNU Affero General Public License 3.0](https://www.gnu.org/licenses/agpl-3.0.en.html). The complete source code for the versions of MinIO packaged with Kaptain 1.2.0 are available at these URLs:
```
